website/docs/cdktf/cli-reference/commands.html.md

```diff
@@ -380,39 +388,42 @@ Synthesize code when providing a custom command to execute and an output directo
 $ cdktf synth --app="npm compile && node main.js" --output="dirname"
 ```
 
-Synthesize code for a given application and output the Terraform JSON configuration.
+Synthesize code for an application and output the Terraform JSON configuration.
 
 ```bash
 $ cdktf synth --json
 ```
 
 ## watch
 
-~> **Warning:** The `watch` command is experimental. You should use it in development environments.
+~> **Warning:** The `watch` command is experimental. You should only use it in development environments.
 
-The `watch` command watches a directory for changes and automatically synthesizes and deploys changes as they happen. It allows for rapid iterations when developing infrastructure, especially when working with serverless services. It currently supports only one stack at a time.
+The `watch` command watches a directory for changes and automatically synthesizes and deploys changes as they happen. It allows for rapid iterations when developing infrastructure, especially when working with serverless services. It currently supports only one stack at a time and automatically deploys changes without asking for confirmation.
```

Comment:
just curious @danieldreier but is there any flag for using `watch` that will prevent it from deploying changes without asking for permission? That seems like something maybe worth calling out with a *warning*?

Reply:
yeah you know what this is on me - I combined this in with the text and now users may miss it.... I'm going to try combining it with the warning we already have above since we probably want to avoid a bunch of warning blocks.  But @danieldreier let us know about the flag as well - if there is one, I can add that too!